cachi2/core/models/sbom.py

```diff
@@ -65,14 +65,17 @@ def from_package_dict(cls, package: dict[str, Any]) -> "Component":
 class Tool(pydantic.BaseModel):
     """A tool used to generate the SBOM content."""
 
-    vendor: str
+    type: str
     name: str
+    author: str
 
 
 class Metadata(pydantic.BaseModel):
     """Metadata field in a SBOM."""
 
-    tools: list[Tool] = [Tool(vendor="red hat", name="cachi2")]
+    tools: dict[str, list[Tool]] = {
+        "components": [Tool(type="application", author="red hat", name="cachi2")]
```

Comment:
components is a list of `Component` per the CycloneDX spec, should we get rid of the `Tool` object?

https://cyclonedx.org/docs/1.5/json/#metadata_tools_oneOf_i0_components

Reply:
Not needed anymore since we'll not update Cachi2's SBOM version.